apps/coordinator/src/components/Wallet/TransactionAnalysis/SWASlider.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { useState, useEffect, useMemo } from "react";
+import { Box, Typography, Slider, Tooltip, IconButton } from "@mui/material";
+import InfoIcon from "@mui/icons-material/Info";
+import { useSelector } from "react-redux";
+import { bitcoinsToSatoshis } from "@caravan/bitcoin";
+import { getWalletConfig, WalletState } from "../../../selectors/wallet";
+import { calculateWasteMetric } from "@caravan/health";
+import BigNumber from "bignumber.js";
+import "../styles.css";
+
+interface TransactionState {
+  fee: number;
+  feeRate: number;
+  outputs: any[];
+  inputs: any[];
+  changeAddress: string;
+}
+
+interface SpendState {
+  transaction: TransactionState;
+}
+
+interface RootState {
+  wallet: WalletState;
+  spend: SpendState;
+}
+
+export const SWASlider = () => {
+  const [longTermFeeEstimate, setLongTermFeeEstimate] = useState<number>(101);
+  const [wasteAmount, setWasteAmount] = useState<BigNumber>(new BigNumber(0));
+
+  const { fee, feeRate, outputs, inputs } = useSelector(
+    (state: RootState) => state.spend.transaction,
+  );
```

Comment:
Let's initialize with the current fee rate that's been set. 

```suggestion
  const { fee, feeRate, outputs, inputs } = useSelector(
    (state: RootState) => state.spend.transaction,
  );
  const [longTermFeeEstimate, setLongTermFeeEstimate] = useState<number>(feeRate);
  const [wasteAmount, setWasteAmount] = useState<BigNumber>(new BigNumber(0));

```